modules/library/src/main/scala/zio/elasticsearch/highlighting/Highlights.scala

```diff
@@ -0,0 +1,27 @@
+package zio.elasticsearch.highlighting
+
+import zio.Chunk
+import zio.elasticsearch.highlighting.Highlights.HighlightConfig
+import zio.json.ast.Json
+import zio.json.ast.Json.Obj
+
+case class Highlights(fields: Chunk[HighlightField], globalConfig: HighlightConfig = Map.empty) { self =>
+  def toJson: Json = Obj("highlight" -> Obj(configList: _*).merge(fieldsList))
+
+  def withGlobalConfig(configFieldName: String, config: Json): Highlights =
+    self.copy(globalConfig = self.globalConfig.updated(configFieldName, config))
+
+  def withHighlight(fieldName: String, fieldConfig: HighlightConfig = Map.empty): Highlights =
```

Comment:
Do we need `Field` here also? And maybe use `field` instead of `fieldName`. 
Alos, I would use `config` instead of `fieldConfig`.